ansible/tasks/hashbang/main.yml

```diff
@@ -6,37 +6,30 @@
       register: gnupghome
     - name: create root .ssh directory
       file: path=/root/.ssh state=directory mode=0700
-    - name: Import administrator PGP keys from keyserver
+    - name: create hashbang lib directory
+      file: path=/var/lib/hashbang state=directory
+    - name: Copy administrator PGP keys
+      git:
+        repo: "https://github.com/hashbang/keyring"
+        dest: "/var/lib/hashbang/keyring"
+    - name: Find copied admin keys
+      find:
+        paths: /var/lib/hashbang/keyring/admins
+      register: admin_keys
```

Comment:
Replace this with a `with_filetree` on line 28? Example here: https://docs.ansible.com/ansible/2.4/playbooks_loops.html#looping-over-filetrees

Reply:
Looks much better. I'll swap it out when I get home.